web/themes/custom/hoeringsportal/templates/field/field--dynamic-block-field--node-public-meeting-summary.html.twig

```diff
@@ -36,6 +36,14 @@
  * @see template_preprocess_field()
  */
 #}
+{% set pretix_dates_field = drupal_field('field_pretix_dates', 'node', element['#object'].id)[0] %}
+{% set variables = pretix_dates_field['#variables'].data %}
+{% set time_to = pretix_dates_field.time_to %}
+{% set now_timestamp = 'now'|date('U') %}
+{% set registration_deadline = drupal_field('field_registration_deadline', 'node', element['#object'].id)[0].date %}
+{% set registration_deadline_passed = registration_deadline and registration_deadline|date('U') <= now_timestamp %}
+{% set meeting_has_been_held = time_to|date('U') <= now_timestamp %}
+{% set cancelled = drupal_field('field_public_meeting_cancelled', 'node', element['#object'].id)[0]['#markup'] %}
```

Comment:
Two suggestions:

1. Make this a boolean, i.e. `set cancelled = drupal_field(…) is same as(1)`
2. Rename to `meeting_cancelled` for clarity.